utils/send-mail.js

```diff
@@ -0,0 +1,11 @@
+
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SEND_GRID_API_KEY);
+const msg = {
```

Comment:
this is hardcoded&mdash;make it a function where we can pass in `{ to, subject, text, html }`